test.py

```diff
@@ -0,0 +1,17 @@
+import requests
```

Comment:
Whoops! Maybe we don't need this file in here. It's based on the videos test file.